addon/data/set-installed-flag.js

```diff
@@ -10,3 +10,8 @@
 
 unsafeWindow.navigator.testpilotAddon = true;
 unsafeWindow.navigator.testpilotAddonVersion = self.options.version;
+
+self.port.on('detach', function() {
+  unsafeWindow.navigator.testpilotAddon = false;
+  unsafeWindow.navigator.testpilotAddonVersion = null;
```

Comment:
Hmm, weird: [I switched this to delete instead](https://github.com/mozilla/testpilot/pull/1460/commits/349b1b59887dfc2150c9d9f2976ab9e38c1f0489#diff-398f10aebbac83980f6fee6c04815d03R15), which works to change the install state. But, it looks like the key is still present on the `navigator` object. Or, at least `'testpilotAddon' in navigator` is `true` before & after add-on uninstall